cmd/users/main.go

```diff
@@ -108,6 +111,8 @@ const (
 	envAuthTimeout = "MF_AUTH_GRPC_TIMEOUT"
 )
 
+var passRegex = regexp.MustCompile(mainflux.Env(envPassRegex, defPassRegex))
```

Comment:
I have used `MustCompile` because I saw it's used on other places:
https://github.com/mainflux/mainflux/blob/master/users/users.go#L26-L28

Reply:
I would use `Compile` instead of `MustCompile` and check the errors:  https://golang.org/pkg/regexp/#Compile 
You can maybe also fix it in users service in this PR

Reply:
I'd do something like that

```
logger, err := logger.New(os.Stdout, cfg.logLevel)
if err != nil {
     log.Fatalf(err.Error())
}

passRegex, err := regexp.MustCompile(mainflux.Env(envPassRegex, defPassRegex))
if err != nil {
     passRegex, err := regexp.MustCompile(defPassRegex)
}
```